src/kixi/datastore/schemastore/validator.clj

```diff
@@ -54,6 +54,10 @@
   [definition _]
   (conformers/regex? (get-in definition [::ss/schema ::ss/pattern])))
 
+(defmethod resolve-schema "string"
+  [_ _]
+  conformers/-string?)
```

Comment:
Ooooh, still confusing a bit :/ Ok then... if you don't think it needs changing :roll_eyes: :)